include/core/truthTable/truth_table.hpp

```diff
@@ -236,9 +240,27 @@ namespace syrec {
         using CubeMap = std::map<Cube, Cube>;
 
     private:
-        CubeMap cubeMap{};
+        CubeMap           cubeMap{};
+        std::vector<bool> constants;
+        std::vector<bool> garbage;
```

Comment:
these vectors need to be properly maintained throughout the "lifetime" of a TT, i.e., they should be properly resized upon construction and modification to the TT. Adding constants should probably populate the respective vector and maybe there also is some function where it is clear that the added variable is to be considered garbage.